book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Please add the `rentals` variable to the `attr_accessor` instead of the `attr_reader`. That will ensure we get a setter and getter for it.

Reply:
addressed